README.md

```diff
@@ -1 +1,33 @@
-# hw01
\ No newline at end of file
+# Computing for the Social Sciences: Homework 1
+## Angel Aliseda Alonso 
+
+### Background
+My name is Angel Aliseda Alonso, I am a second-year MPP student from Harris School of Public Policy. I was born in Mexico and before coming to the University of Chicago I worked at the Mexican Social Security Institute. My professional interests are __health policy__, __health systems reform__ and __program evaluation__, but I also enjoy __running next to Lake Michigan__, __traveling__ and food (specially __desserts__).
+
+### Education
+
+* B.A. in Economics from the Instituto Tecnologico Autonomo de Mexico (ITAM), _2009-2014_
+* B.A. in Political Science from the Instituto Tecnologico Autonomo de Mexico (ITAM), _2009-2014_
+
+### Work Experience
+
+* __Mexican Social Security Institute (IMSS)__
+  * _Advisor to the Director of Strategic Planning, 2016-2018_
```

Comment:
Nice use of multiple levels in a bullet list. I was trying to figure out how to do the same thing in my own assignment -- I learned something new!